portfolio/src/main/webapp/notes.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Notes</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+
+    <h1>List of notes</h1>
+    <p>Notes/Questions/Comments I had while going through my webpage</p>
+
+    <p>
+        Webpage updates in real time only files that have been previously staged and comitted
+        <br>
+        However, that doesn't apply to unstaged files (a.k.a. have never been staged)
```

Comment:
Are you talking about Git?

Reply:
I meant in the cloud shell itself; if I made changes to a file I had already staged, then I could just F5 the webpage (on port 8080). But as of right now, it's not doing that anymore; I have to run <code>mvn package exec:java</code> each time for the changes to register

Reply:
Oh. I always just re-run the server.